three.js/examples/measure-it.html

```diff
@@ -189,7 +189,7 @@
 		geometry.verticesNeedUpdate = true
 
 		geometry.computeBoundingSphere();
-		geometry.computeLineDistances();
```

Comment:
This is deprecated in Three.js r103, instead we should use THREE.Line.computeLineDistances()